travis-ci/vmtest/run_selftests.sh

```diff
@@ -7,15 +7,13 @@ source $(cd $(dirname $0) && pwd)/helpers.sh
 STATUS_FILE=/exitstatus
 
 test_progs() {
-	if [[ "${KERNEL}" != '4.9.0' ]]; then
-		travis_fold start test_progs "Testing test_progs"
-		# "&& true" does not change the return code (it is not executed
-		# if the Python script fails), but it prevents exiting on a
-		# failure due to the "set -e".
-		./test_progs ${BLACKLIST:+-b$BLACKLIST} ${WHITELIST:+-t$WHITELIST} && true
-		echo "test_progs:$?" >> "${STATUS_FILE}"
-		travis_fold end test_progs
-	fi
+  travis_fold start test_progs "Testing test_progs"
+  # "&& true" does not change the return code (it is not executed
+  # if the Python script fails), but it prevents exiting on a
+  # failure due to the "set -e".
+  ./test_progs ${BLACKLIST:+-b$BLACKLIST} ${WHITELIST:+-t$WHITELIST} && true
+  echo "test_progs:$?" >> "${STATUS_FILE}"
+  travis_fold end test_progs
```

Comment:
looks like indentation is off?

Reply:
github doesn't display tab characters well, I've switched everything back to space indent